app/templates/gentelella/guest/ticketing/components/_order_summary.html

```diff
@@ -1,5 +1,5 @@
 <div id="order-summary-holder">
-    <div class="panel panel-default">
+    <div class="panel panel-default" style="margin-top: 20px">
```

Comment:
Adding `margin-top: 20px` breaks the design in pre_payment page.
![screenshot from 2017-01-18 09-44-59](https://cloud.githubusercontent.com/assets/9530293/22050679/f105ee6a-dd62-11e6-9f32-41767911b88e.png)

The 2 boxes side-by-side should begin from the same level. 

Firstly the overlap was made consciously. Secondly, even if I consider that as a bug, this doesn't fix the bug.